File: dev-docs/snippets/collections/remove-document.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Remove a document from a collection. Returns `{ success: boolean }`.
export async function removeDocument(
  client: JsBaoClient,
  collectionId: string,
  documentId: string,
) {
  // #region example
  const { success } = await client.collections.removeDocument(
    collectionId,
    documentId,
  );
  // #endregion example
  return success;
}

File: dev-docs/snippets/collections/remove-document.swift
import JsBaoClient

// Remove a document from a collection. Returns `SuccessResult { success }`.
func removeDocument(
  client: JsBaoClient,
  collectionId: String,
  documentId: String
) async throws {
  // #region example
  let result = try await client.collections.removeDocument(
    collectionId: collectionId,
    documentId: documentId
  )
  let success = result.success
  // #endregion example
  _ = success
}
